crud/select/merger.lua

```diff
@@ -230,6 +230,68 @@ local function new(vshard_router, replicasets, space, index_id, func_name, func_
     return merger
 end
 
+
+local function new_readview(vshard_router, replicasets,readview_uuid, space, index_id, func_name, func_args, opts)
+    opts = opts or {}
+    local call_opts = opts.call_opts
+    local mode = call_opts.mode or 'read'
+    local vshard_call_name = call.get_vshard_call_name(mode, call_opts.prefer_replica, call_opts.balance)
```

Comment:
These are also irrelevant for a call since we already know exactly which one we expect to call.

Reply:
changed